py/test/selenium/webdriver/marionette/mn_launcher_tests.py

```diff
@@ -16,6 +16,7 @@
 # under the License.
 
 from selenium.webdriver import Firefox
+from selenium.webdriver.firefox.service import Service
```

Comment:
`import pytest` is missing i think

Reply:
True. Done.
wish I had some way to run the tests so I could see if I missed anything else.